app/services/tag_service.rb

```diff
@@ -0,0 +1,17 @@
+class TagService
```

Comment:
W tym wypadku bedzie to raczej form object. Problem polega na tym że powoduje to rozsianie tego jak działa `tag_list` po całej aplikacji i potem ciężko to testować. Dlatego sądzę że najlepiej jak to siedzi w modelu - jedno obok drugiego, bez żadnych udziwnień.